src/server/debugcmd.cc

```diff
@@ -536,8 +536,9 @@ void DebugCmd::Load(string_view filename) {
   }
 
   auto new_state = sf_.service().SwitchState(GlobalState::ACTIVE, GlobalState::LOADING);
-  if (new_state.first != GlobalState::LOADING) {
-    LOG(WARNING) << GlobalStateName(new_state.first) << " in progress, ignored";
+
+  if (new_state != GlobalState::LOADING) {
+    LOG(WARNING) << GlobalStateName(new_state) << " in progress, ignored";
```

Comment:
```suggestion
    LOG(WARNING) << new_state << " in progress, ignored";
```
Isn't this why you added the above `operator<<`?